keyboards/the_royal/schwann/config.h

```diff
@@ -0,0 +1,40 @@
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x1337
```

Comment:
This Vendor ID is owned, please change it.

Reply:
I never understood that before lol. 

Ive never changed or messed with that config in the past